client/memory.js

```diff
@@ -79,12 +79,13 @@ export default class Memory extends React.Component {
       }
     }).then(function(res) {
       var memory = JSON.parse(res['_bodyInit']);
+      console.log(memory);
       var microsoftTags = [];
       var clarifaiTags = [];
-      if (memory.analyses[0] === null) {
+      if (memory.analyses[0].tags && memory.analyses.tags.length > 0) {
         microsoftTags = memory.analyses[0].tags;
```

Comment:
I'm not sure if the above comment matters or not. 

One more consideration in the design of this code is scalability. If we were to integrate with 10 APIs on the backend, would we have to manually search for the tags from each of those APIs? Would it make sense to do something like 

var allTags = [];
memory.analyses.forEach((analysis) => { if (analysis.tags) { allTags = allTags.concat( analysis.tags ); }};